Ports/openssl/package.sh

```diff
@@ -3,17 +3,17 @@ port=openssl
 version=1.0.2t
 useconfigure=true
 configscript=Configure
-files="https://www.openssl.org/source/openssl-${version}.tar.gz openssl-${version}.tar.gz
-https://www.openssl.org/source/openssl-${version}.tar.gz.asc openssl-${version}.tar.gz.asc"
+files="https://ftp.openssl.org/source/old/1.0.2/openssl-${version}.tar.gz openssl-${version}.tar.gz
```

Comment:
Should the `1.0.2` part here be `${version}` or something instead? Or is this the right value for all versions?

Reply:
Thanks, I did miss this.

(I tried to compile `1.1.0l` quickly using the same patches, but they need rework for the newer branch).